src/App.tsx

```diff
@@ -1,28 +1,69 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { FilterStatusType, Todo } from './types/Todo';
+import * as tadoService from './api/todos';
+import { Footer } from './component/Footer';
+import { Error } from './component/Error';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { TodoItem } from './component/Todo';
+
+function filterTodos(todoArr: Todo[], filter: FilterStatusType) {
+  const todos = [...todoArr];
```

Comment:
- Move to utils/helpers file/folder
- `filter` doesn't mutate an array so you don't need to create a copy